modules/gitops-release-notes-1-15-0.adoc

```diff
@@ -28,10 +28,93 @@ If you have installed the {gitops-title} Operator in the default namespace, run
++
+When you enable high availability, the operator automatically sets the number of replicas to 2, the leader election is activated, and default anti-affinity rules are applied to ensure redundancy and reliability.
+
+* With this update, the Argo CD multi-source application functionality in {gitops-shortname} Operator is promoted from Technology Preview (TP) to General Availability (GA). Use this feature to define and manage applications with multiple source repositories. link:https://issues.redhat.com/browse/GITOPS-5511[GITOPS-5511]
+
+* With this update, you can now use Kubernetes TLS-type secrets to configure custom TLS certificates for Routes in the Argo CD CR. Previously, TLS data had to be embedded directly in the Argo CD CR as plain text, which could pose security risks due to the sensitive nature of TLS information.
++
+To enhance security, you must use Kubernetes secrets to manage TLS data. A new field, `.spec.route.tls.externalCertificate`, has been introduced in the Argo CD CR to reference Kubernetes secrets of type `kubernetes.io/tls` containing TLS data. For more information about configuring a route for custom TLS certificate, see examples in link:https://argocd-operator.readthedocs.io/en/latest/usage/routes/#custom-tls-certificates[Custom TLS certificates for Routes]. link:https://issues.redhat.com/browse/GITOPS-5232[GITOPS-5232]
+
+* With this update, Restricted pod security admission (PSA) labels are applied to the `openshift-gitops` namespace to ensure compliance with {OCP} standards. If you are running additional workloads in the `openshift-gitops` namespace, ensure they comply with restricted PSA requirements. If compliance with restricted PSA is not feasible, consider using a user-defined, cluster-scoped Argo CD instance, where PSA labels are not applied or controlled by the {gitops-shortname} Operator in these instances. link:https://issues.redhat.com/browse/GITOPS-5221[GITOPS-5221]
+
+* Before this update, `trafficmanagement` and `metric` plugins could not be configured by using the `RolloutManager` CR in Argo Rollouts. With this update, you can now specify `trafficmanagement` and `metric` plugins in the `RolloutManager` CR. The {gitops-shortname} Operator automatically updates the config map when these plugin fields are added, modified, or removed. This enhancement allows users to manage `trafficmanagement` and `metric` plugins natively through the `RolloutManager` CR, eliminating the need for manual config map updates. link:https://issues.redhat.com/browse/GITOPS-4217[GITOPS-4217]
+
+* With this update, the *Sources* tab in the Details panel of a multi-source application now includes two new options:
+** *Add*: Launches a form to add a new source entry to the application. You can specify the repository URL and other source-specific details.
```

Comment:
If you change it to 'buttons' in my above comment, then this needs to change from "*Add*" to "*Add Source*" because that is the actual name of the button.